app/main.cpp

```diff
@@ -77,6 +79,12 @@ int main(int argc, char *argv[])
     app.setApplicationName(QLatin1String("perfparser"));
     app.setApplicationVersion(QLatin1String("4.5"));
 
+    if (qEnvironmentVariableIsSet("PERFPARSER_DEBUG_WAIT")) {
```

Comment:
why not simply run `perfparser` outside of hotspot to debug it? or alternatively use https://sourceware.org/gdb/onlinedocs/gdb/Forks.html?

Reply:
`set follow-fork-mode child` looks useful, when running with GDB. However, in an IDE like QtCreator, there is no easy way to set that.
Simply running `perfparser` outside of hotspot requires knowing the exact command line, I find it more convenient to just attach to it.